src/ArrowFunctions.cpp

```diff
@@ -685,7 +737,7 @@ int cpp_createEmptyParquetFile(const char* filename, const char* dsetname, int64
 
 int cpp_appendColumnToParquet(const char* filename, void* chpl_arr,
                               const char* dsetname, int64_t numelems,
-                              int64_t dtype, bool compressed,
+                              int64_t dtype, int64_t compression,
```

Comment:
does this function ever use the compression argument?

Reply:
This is due to how it is called from the message parsing. I believe we are deprecating in the future, so did not want to alter too much.